exercises/chapter11/src/Main.purs

```diff
@@ -1,27 +1,29 @@
 module Main where
 
 import Prelude
-import Effect (Effect)
-import Node.ReadLine as RL
+
+import Control.Monad.RWS (RWSResult(..), runRWS)
 import Data.Either (Either(..))
 import Data.Foldable (for_)
 import Data.GameEnvironment (GameEnvironment, gameEnvironment)
 import Data.GameState (GameState, initialGameState)
 import Data.Maybe (Maybe(..))
 import Data.Newtype (wrap)
 import Data.String (split)
+import Effect (Effect)
+import Effect.Console (log)
```

Comment:
Assuming we want `log` from `Effect.Console`, even though importing from `Effect.Class.Console` works too.